Assets/Scripts/Steam/SteamManager.cs

```diff
@@ -414,8 +414,16 @@ public void RequestAppTicket(Action<string> callback)
         /// NOTE: This conditional is here because if EOS_DISABLE is enabled, the members referenced
         ///       in this code block will not exist on EOSManager.
 #if !EOS_DISABLE
-        public async void StartLoginWithSteam(EOSManager.OnAuthLoginCallback onLoginCallback)
-        {
+        public
+            // NOTE: This conditional is here because the function should only
+            //       be marked as 'async' if it calls an awaitable function 
+            //       using the 'await' keyword. When SteamWorks is disabled, 
+            //       there is no such call, which results in a compiler warning.
+#if !DISABLESTEAMWORKS
+            async 
+#endif
+            void StartLoginWithSteam(EOSManager.OnAuthLoginCallback onLoginCallback)
```

Comment:
It might be better to have an empty `await` call within this - so that the `async` keyword is not conditionally applied - which _might_ cause some weird behavior - but more to the point - it will be clearer in the code what is happening if we await an empty thing.

Reply:
Changes have been made as suggested above, marking conversation as resolved.